### ISSUE TITLE
Build failed in Remix app when using @casl/prisma

### ISSUE THREAD

**Issue by revickulous2001** [stalniy/casl#698]
**Describe the bug**
Build failed when using `@casl/prisma` in a Remix app due to importing from "./runtime" instead of "./runtime.mjs"

```
Could not resolve "./runtime"
node_modules/@casl/prisma/dist/es6m/index.mjs:1:154:
      1 │ ... as o,prismaQuery as e}from"./runtime";export{ParsingQueryError,...
```

**To Reproduce**
1) Install `@casl/ability` and `@casl/prisma` in a remix run app.  I used the Indie Stack starter template.
2) Build the app using the `npm run dev` script.
3) The error occurs on build.

**Expected behavior**
A clean build.

By modifying the file `node_modules/@casl/prisma/dist/es6m/index.mjs` to change all `import * from "./runtime"` to `import * from "./runtime.mjs"`, I was able to get a clean and working build.

**CASL Version**

`@casl/ability` - v6.3.2
`@casl/prisma` -v1.3.1

**Environment**:
`node` -v18.4.0

**Comment by jlwbr** [comment]
This patch can be used with[patch-package](https://github.com/ds300/patch-package) as a temporary fix

```diff
diff --git a/node_modules/@casl/prisma/dist/es6m/index.mjs b/node_modules/@casl/prisma/dist/es6m/index.mjs
index 793f2c1..af2ec67 100644
--- a/node_modules/@casl/prisma/dist/es6m/index.mjs
+++ b/node_modules/@casl/prisma/dist/es6m/index.mjs
@@ -1,2 +1,2 @@
-import{PureAbility as r,fieldPatternMatcher as t}from"@casl/ability";import{createAbilityFactory as i,createAccessibleByFactory as o,prismaQuery as e}from"./runtime";export{ParsingQueryError,prismaQuery}from"./runtime";const s=i();const c=o();class PrismaAbility extends r{constructor(r,i){super(r,Object.assign({conditionsMatcher:e,fieldMatcher:t},i))}}export{PrismaAbility,c as accessibleBy,s as createPrismaAbility};
+import{PureAbility as r,fieldPatternMatcher as t}from"@casl/ability";import{createAbilityFactory as i,createAccessibleByFactory as o,prismaQuery as e}from"./runtime.mjs";export{ParsingQueryError,prismaQuery}from"./runtime.mjs";const s=i();const c=o();class PrismaAbility extends r{constructor(r,i){super(r,Object.assign({conditionsMatcher:e,fieldMatcher:t},i))}}export{PrismaAbility,c as accessibleBy,s as createPrismaAbility};
 //# sourceMappingURL=index.mjs.map
```

**Comment by soulchild** [comment]
I'm seeing the same after migrating from `jest` to `vitest`

**Comment by stalniy** [comment]
fixed in @casl/prisma@1.3.3

**Comment by soulchild** [comment]
Works like a charm. Thanks for the quick fix!
